core/src/impl/Kokkos_AnalyzePolicy.hpp

```diff
@@ -49,19 +49,35 @@
 #include <Kokkos_Concepts.hpp>
 #include <impl/Kokkos_Tags.hpp>
 #include <impl/Kokkos_GraphImpl_fwd.hpp>
+#include <impl/Kokkos_Error.hpp>
+#include <type_traits>
+
 namespace Kokkos {
-namespace Impl {
+namespace Experimental {
+struct DesiredOccupancy {
+  int _occ = 100;
+  explicit DesiredOccupancy(int occ) : _occ(occ) {
+    KOKKOS_EXPECTS(0 <= occ && occ <= 100);
+  }
+  explicit operator int() const { return _occ; }
+  DesiredOccupancy() = default;
+};
+struct MaximizeOccupancy {};
```

Comment:
I don't see a way to `prefer()` `MaximizeOccupancy`. Even if it's the default, you should have the overload (or just inherit from `DesiredOccupancy` with a value of `100`?)

Reply:
I added the overload.